Makefile

```
@@ -178,3 +178,21 @@ build-webhook:
 .PHONY: docker-build-webhook
 docker-build-webhook:
 	docker build -f cmd/admission-webhook-server/Dockerfile -t $(ADMISSION_WEBHOOK_SERVER) .
+
+.PHONY: check-update-assets
+check-update-assets:
+	@test -z "$$(git status --porcelain)" || (echo "Please run make update-assets and commit the changes.";git --no-pager diff; exit 1)
+	make update-assets
```

We need to do both. Like:
```suggestion
	@test -z "$$(git status --porcelain)" || (echo "Commit all changes before running this target"; exit 1)
	make update-assets
	@test -z "$$(git status --porcelain)" || (echo "Please run make update-assets and commit the changes.";git --no-pager diff; exit 1)
```